### DISCUSSION TITLE
Using selectors in ControlTheme to change a property on one control based on the property of another

### DISCUSSION THREAD

**Discussion by andy-westacott** [AvaloniaUI/Avalonia#15781] [Q&A]
Hi 
I'm working on a combobox type control which is going fairly well.

One detail I want to handle.
I have a Border portion of the control that I want to modify only when the popups IsOpen=true.
I was hoping to do this with Selectors but I'm having some trouble

The border is 
```
<Border Name="PART_Border" BorderThickness="0.5,0.5,0.5,0.5"  CornerRadius="3,3,3,3">
```
Then the popup
```
<Popup Name="PART_Popup"/>
```
and I was hoping something like 
```
			<Style Selector="^[IsOpen=true]/template/ Popup#PART_Popup">
				<Setter Property="SomeBorderProperty" Value="True" />
			</Style>
```
would work.  This is obviously wrong.

Any pointers to how I would achieve this?

**Comment by timunie** [comment]
at least a missing white space in the selector.

**Reply by andy-westacott** [reply]
Yeah noticed that.  So I have this now which compiles and runs but has no effect

> 			<Style Selector="^Popup[IsOpen=true] /template/ Border#PART_Border">
> 				<Setter Property="BorderBrush" Value="Red" />
> 			</Style>

**Comment by maxkatz6** [comment]
1. Border needs to be inside of the Popup. Neighboring selectors are not supported.
2. Your border is not part of the Popup template (I assume). So, you need to use child selector ">" instead of "/template/".
